cluster/gce/util.sh

```diff
@@ -853,6 +873,80 @@ function kube-down {
   set -e
 }
 
+# Checks if there are any present resources related kubernetes cluster.
+#
+# Assumed vars:
+#   MASTER_NAME
+#   NODE_INSTANCE_PREFIX
+#   ZONE
+# Vars set:
+#   KUBE_RESOURCE_FOUND
+
+function check-resources {
+  detect-project
+
+  echo "Looking for already existing resources"
+  KUBE_RESOURCE_FOUND=""
+
+  if gcloud preview managed-instance-groups --project "${PROJECT}" --zone "${ZONE}" describe "${NODE_INSTANCE_PREFIX}-group" &>/dev/null; then
+    KUBE_RESOURCE_FOUND="Managed instance group ${NODE_INSTANCE_PREFIX}-group"
+    return 1
+  fi
+
+  if gcloud compute instance-templates describe "${NODE_INSTANCE_PREFIX}-template" &>/dev/null; then
```

Comment:
In fact, if you copied `kube-down`, please please fix those. :( Oi it's leaking a bunch....